virttest/libvirt_xml/domcapability_xml.py

```diff
@@ -35,7 +35,8 @@ def __init__(self, virsh_instance=base.virsh):
         result = self.__dict_get__('virsh').domcapabilities()
         self['xml'] = result.stdout_text.strip()
 
-    def get_additional_feature_list(self, cpu_mode_name):
+    def get_additional_feature_list(self, cpu_mode_name,
+                                    ignore_features=("invtsc",)):
```

Comment:
@Yingshun As tuples are immutable, I'd rather have a list here because I expect it to be easier to construct a list of `ignore_features` than a tuple. But I might be wrong.

Reply:
@smitterl  Hmm, that's the reason why I use a tuple type. I need to set an argument that includes 'invtsc' as a default value for consistency, in other words, I can't set it to None as a default value. In this case, using a list is not recommended, the value of default value would be changed if someone wrote 'append' function.

Example:
```
>>> def mytest(mylist=["invtsc"]):
...  mylist.append("ds")
...  print(mylist)
... 
>>> mytest()
['invtsc', 'ds']
>>> mytest()
['invtsc', 'ds', 'ds']
>>> mytest()
['invtsc', 'ds', 'ds', 'ds']
>>> 

```